Modules/LootManager/GUI.lua

```diff
@@ -142,12 +143,15 @@ local function CreateLootDisplay(self)
     self.st:SetFilter((function(stobject, row)
         local item = strlower(ST_GetItemLink(row))
         local searchKeyword = self.SearchInput:GetText()
-        if searchKeyword and searchKeyword ~= "" and strlen(searchKeyword) >= 3 then
+        if searchKeyword and strlen(searchKeyword) >= 3 then
```

Comment:
Why not check for trimmed string length here? You are now running some lines of code that makes no sense if someone just enters some spaces

Reply:
We need to detect if anything is input there and there is no computational difference if we check >0 or >N so i used 3 as this is the minimal chars we do search for in a searched entry

Reply:
discussed -> Trim is not performant because of RegEx check